app/config/ezplatform.yml

```diff
@@ -37,6 +38,9 @@ ezpublish:
               # As we by default enable EzSystemsPlatformHttpCacheBundle which is designed to expire all affected cache
               # on changes, and as error / redirects now have separate ttl, we easier allow ttl to be greatly increased
               default_ttl: '%httpcache_default_ttl%'
-            # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
```

Comment:
maybe skip this diff to avoid conflicts on future merges

Reply:
ok, removed